internal/services/instance_types_service_test.go

```diff
@@ -55,15 +58,15 @@ func TestListInstanceTypesHandler(t *testing.T) {
 		ctx = clientStub.WithEC2Client(ctx)
 
 		req, err := http.NewRequestWithContext(ctx, "GET", "/api/provisioning/sources/1/instance_types", nil)
-		require.NoError(t, err, "failed to create request")
+		assert.Nil(t, err, fmt.Sprintf("Error creating a new request: %v", err))
 
 		rr := httptest.NewRecorder()
 		handler := http.HandlerFunc(services.ListInstanceTypes)
 		handler.ServeHTTP(rr, req)
 
 		require.Equal(t, http.StatusBadRequest, rr.Code, "Handler returned wrong status code")
 
-		assert.Error(t, payloads.ParamMissingError{})
+		assert.Contains(t, rr.Body.String(), "missing parameter")
```

Comment:
Seems like that except this change, this reverts my changes from #210 is that intentional?

Reply:
Nah, this was a bad conflict merge. I meant to change only this line. Fixed.